google-cloud-language/lib/google/cloud/language/annotation.rb

```diff
@@ -241,8 +241,8 @@ def self.from_grpc grpc
         #
         # @attr_reader [String] text The content of the output text.
         # @attr_reader [Integer] offset The API calculates the beginning offset
-        #   of the content in the original document according to the `encoding`
-        #   specified in the API request.
+        #   of the content in the original document according to the default
+        #   encoding of the system. Ruby defaults to UTF-8.
```

Comment:
Do you want to add:

> To change this you will need to set `Encoding.default_internal` to `Encoding ::UTF_16` or `Encoding ::UTF_32`.

Otherwise, you have to hope that someone interested in the offset is going to stumble upon this in the guide or Document docs.